src/code/client.js

```diff
@@ -561,24 +561,36 @@ class CloudFileManagerClient {
       saving: metadata})
     const currentContent = this._createOrUpdateCurrentContent(stringContent, metadata)
     return metadata.provider.save(currentContent, metadata, (err, statusCode) => {
+      let failures;
       if (err) {
-        // disable autosave on save failure; clear "Saving..." message
-        metadata.autoSaveDisabled = true
-        this._setState({ metadata, saving: null })
+        this._setState({ metadata, saving: null });
         if (statusCode === 403) {
-          return this.confirmAuthorizeAndSave(stringContent, callback)
+          return this.confirmAuthorizeAndSave(stringContent, callback);
         } else {
-          return this.alert(err)
+          failures = this.state.failures
```

Comment:
```suggestion
          failures = this.state.failures;
```
It appears that we're inconsistent on commas.

Reply:
Upon further review it appears that _not_ having semicolons is now the project default, so I'm going to go ahead and remove them here.